tests/templates/kuttl/smoke/10-install-opa.yaml.j2

```diff
@@ -33,6 +33,12 @@ spec:
     config:
       logging:
         enableVectorAgent: {{ lookup('env', 'VECTOR_AGGREGATOR') | length > 0 }}
+      resources:
+        cpu:
+          max: '1'
+          min: '300m'
+        memory:
+          limit: '512Mi'
```

Comment:
Can we test the default values here to make sure they are enough to pass the smoke tests?

Reply:
https://github.com/stackabletech/opa-operator/pull/453/commits/0bb646390e50a01908b7097eddec8b4fc78af27a